index.html

```diff
@@ -0,0 +1,671 @@
+          </div>
+          <h2>San Francisco Residents Share Videos, Photos Of 'Apocalyptic' Skies As Wildfires Burn Nearby</h2>
+          <p>The afternoon skies were bright orange in San Francisco on Wednesday as nearby wildfires continue to blaze across the state.</p>
+        </article>
+        <div class="col-title">MY TURN</div>
+        <section class="fluid-container">
+          <div class="row">
+            <article class="story-blocks d-flex flex-column col-4">
+              <img alt="Article Image" src="img/story-1.jpg">
+              <p class="story-title">'I'm a Marriage Therapist, Here Are 3 Ways Marriage Has Changed' </p>
+            </article>
+            <article class="story-blocks d-flex flex-column col-4">
+              <img alt="Article Image" src="img/story-1.jpg">
+              <p class="story-title">'I'm a Marriage Therapist, Here Are 3 Ways Marriage Has Changed' </p>
+            </article>
+            <article class="story-blocks d-flex flex-column col-4">
+              <img alt="Article Image" src="img/story-1.jpg">
+              <p class="story-title">'I'm a Marriage Therapist, Here Are 3 Ways Marriage Has Changed' </p>
+            </article>
+          </div>
+        </section>
+        <div class="col-title">CULTURE AND TRAVEL</div>
+        <section class="fluid-container">
+          <div class="row">
+            <article class="story-blocks d-flex flex-column col-4">
+              <div class="img-w-sub">
+                <div class="inner-subtitle d-block">
+                  <a href="#">TRAVEL</a>
+                </div>
+                <img alt="Article Image" class="img-fluid" src="img/culture-story-1.jpg">
+              </div>
+              <p class="story-title">The Countries With the Least Amount of Travel Buzz, According to Google </p>
+            </article>
+            <article class="story-blocks d-flex flex-column col-4">
+              <div class="img-w-sub">
+                <div class="inner-subtitle d-block">
+                  <a href="#">TRAVEL</a>
+                </div>
+                <img alt="Article Image" class="img-fluid" src="img/culture-story-1.jpg">
+              </div>
+              <p class="story-title">The Countries With the Least Amount of Travel Buzz, According to Google </p>
+            </article>
+            <article class="story-blocks d-flex flex-column col-4">
+              <div class="img-w-sub">
+                <div class="inner-subtitle d-block">
+                  <a href="#">TRAVEL</a>
+                </div>
+                <img alt="Article Image" class="img-fluid" src="img/culture-story-1.jpg">
+              </div>
+              <p class="story-title">The Countries With the Least Amount of Travel Buzz, According to Google </p>
+            </article>
+          </div>
+        </section>
+      </div>
+
+      <div class="col3 container">
+        <div class="col-title">THE DEBATE</div>
```

Comment:
[OPTIONAL] `MORE STORIES` section is missing. This is one of the required sections for this project. However, since it wasn't mentioned previously, I'll be marking it as optional. Kindly fix this.